test/annealing/metal_test.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+describe Annealing::Metal do
```

Comment:
Same here, the test suite must be cosiste with the same idiom. A plain ruby object for tests is more than enough